src/theme/variables.ts

```diff
@@ -99,6 +99,10 @@ export const baseVariables = css`
     --ion-background-color: var(--ion-color-step-50, #f2f2f7);
     --ion-item-background: #fff;
   }
+
+  ion-icon {
```

Comment:
Awesome!

Can you move this to index.css?

Sorry the naming is confusing, TBD

Reply:
I moved it and mentioned the ionicon issue in a comment. Since it's just a workaround it can be removed once/if the original issue is resolved. Thought that might help a bit.

Reply:
Thanks very much @fer0n!